fastestimator/summary/system.py

```diff
@@ -43,31 +43,40 @@ def __init__(self,
 
         self.network = network
         self.mode = mode
-        self.global_step = 0
+        self.global_step = None
```

Comment:
what is the point of None here? if not absolutely necessary better to avoid, and if you do change, need to change the typing info above to Optional[int]

Reply:
with None, it's easier to tell whether the global step is initialized or not.    fixed the typing.